Gopkg.toml

```diff
@@ -116,3 +116,15 @@
 [[constraint]]
   name = "github.com/hashicorp/go-plugin"
   branch = "master"
+
+[[constraint]]
+  name = "github.com/golang/protobuf"
+  branch = "master"
```

Comment:
let's pin this to a specific semver that's compatible with the go-plugin commit - so instead of `branch = "master"`, `version = "~v1.3.1"`